src/main/java/org/spongepowered/api/world/volume/game/TrackedVolume.java

```diff
@@ -98,7 +103,13 @@ default void setCreator(Vector3i pos, @Nullable UUID uuid) {
      * @param z The z coordinate where the user data should be applied
      * @param uuid The {@link UUID} to set as creator
      */
-    void setCreator(int x, int y, int z, @Nullable UUID uuid);
+    default void setCreator(final int x, final int y, final int z, @Nullable final UUID uuid) {
```

Comment:
I disagree with these defaults. I always thought that a particular volume holder may have a faster way to query this data instead of going through a provider.

Granted this is a default so not like we can't do that..

Reply:
It's a default because the Keys exist and it's often better to delegate to those, however, implementation wise they serve the purpose of bypassing the requirement that we re-implement the actual storage of owner/notifier at this time.